spring-security-oauth/src/main/java/org/springframework/security/oauth/common/OAuthParameter.java

```diff
@@ -17,11 +17,12 @@
 package org.springframework.security.oauth.common;
 
 /**
- * Enumeration for consumer parameters.
+ * OAuth parameters.
  *
  * @author Ryan Heaton
+ * @author <a rel="author" href="http://autayeu.com/">Aliaksandr Autayeu</a>
  */
-public enum OAuthConsumerParameter {
```

Comment:
... It seems that there is a place where we actually need this to be an Enum, they were actually iterating through the Enum values in class `org.springframework.security.oauth.consumer.client.CoreOAuthConsumerSupport` line 365, so disregard my previous comment.